Library/Formula/dwarfutils.rb

```diff
@@ -0,0 +1,28 @@
+class Dwarfutils < Formula
+  desc "lib and utility to dump and produce DWARF debug info in ELF objects"
+  homepage "http://www.prevanders.net/dwarf.html"
+  url "http://www.prevanders.net/libdwarf-20150507.tar.gz"
+  sha256 "29aa8d07db659d7d7af7079854cf42c09bf74d303942159cbfee82d655549870"
+
+  depends_on "libelf" => :build
+
+  def install
+    # Disable building the shared library, since the package does not understand
+    # OS X naming quirks (dylib extension) and rpath quirks, and the static
+    # library works just fine.
+    system "./configure", "--disable-shared"
```

Comment:
I'll look into it, but since the static library works with so little trouble, and it useful in and of itself, is it okay to merge this formula with the shared library disabled, and update it if it gets fixed upstream?

FWIW I think `--disable-shared` is the upstream default anyways, I was just being explicit here and explaining why the shared library isn't here for anyone who wondered wandering by.